src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java

```diff
@@ -8359,12 +8359,16 @@ public static Value last_monster(ScriptRuntime controller) {
     return DataTypes.makeMonsterValue(MonsterStatusTracker.getLastMonster());
   }
 
-  private static MonsterData mapMonster(MonsterData mon, Map<MonsterData, MonsterData> mapping) {
-    if (mapping == null) {
+  private static MonsterData mapMonster(
+      MonsterData mon,
+      Map<MonsterData, MonsterData> classMapping,
+      Map<MonsterData, MonsterData> pathMapping) {
+    if (classMapping == null && pathMapping == null) {
```

Comment:
if either of these two are null, it will still NPE.

I would check for a path mapping first, as almost all the time there won't be a class mapping.

Reply:
Right. Fixed.